backend/graph/api/schema/schema.graphqls

```diff
@@ -120,6 +120,10 @@ type QueryRoot{
 
   me: User!
 
+  myList: UserCollection!
```

Comment:
Do we only have 1 list? Or is this a special/default one?
Specific per app & profile?

Reply:
At the moment it's only 1 list. 

I wanted the same structure though in case we want users to be able to create multiple lists down the line.

Reply:
Actually filtered by a parameter in the metadata field, so only 1 per profile & app.